generator/src/main/java/com/scottlogic/deg/generator/generation/IsinStringGenerator.java

```diff
@@ -222,14 +222,14 @@ public boolean match(String subject) {
 
     private static StringGenerator getIsinSansCheckDigitGeneratorForCountry(String countryCode) {
         if (countryCode.equals("GB")) {
-            return new SedolStringGenerator(countryCode);
+            return new SedolStringGenerator("GB00");
```

Comment:
would it be better to add the "00" to the SedolStringGenerator constructor, rather than here?

Reply:
Not really, because it's not part of the SEDOL itself.  The SedolStringGenerator shouldn't know about it, because it shouldn't need to know what its SEDOLs are being used for.